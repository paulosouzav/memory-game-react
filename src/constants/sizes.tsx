export const SIZES = Object.freeze({
  navHeight: '60px',
});
