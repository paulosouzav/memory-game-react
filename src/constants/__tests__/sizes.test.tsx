import { SIZES } from '../sizes';

describe('SIZES', () => {
  it('should have expected values', () => {
    expect(SIZES).toMatchSnapshot();
  });
});
