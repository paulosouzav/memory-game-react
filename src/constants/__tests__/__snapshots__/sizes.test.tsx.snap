// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SIZES should have expected values 1`] = `
Object {
  "navHeight": "60px",
}
`;
